### ISSUE TITLE
check_syntax: Alternate WA usage for dictionary tables is not allowed in function call

### ISSUE THREAD

**Issue by ed-holland** [abaplint/abaplint#3353]
Linter: abaplint 2.106.5

Abap allows to use an alternate wa for a dictionary table. This is done by putting a "*" in front.
The linter does not allow this. Can it be added?

```
./src/ecc/#jcs#fi.fugr.#jcs#fi_tstor_action.abap[359, 3]                      - "*bkpf" not found, findTop (check_syntax) [E]
```

Source
![image](https://github.com/abaplint/abaplint/assets/53995991/a4f43ae6-ce3e-49d2-b464-3cab3fd5404b)

**Comment by larshp** [comment]
whats in line 359, where the error is reported?

**Comment by ed-holland** [comment]
Sorry, wrong screenshot
![image](https://github.com/abaplint/abaplint/assets/53995991/0b780ab1-e791-4791-9749-786f77669640)

**Comment by ed-holland** [comment]
This issue is also resolved with version 2.111.0
